src/components/mainLayout/signInForm/signInForm.tsx

```diff
@@ -84,6 +86,16 @@ const SignInForm = () => {
 		router.push(UserProfilePath.ProfitMoney);
 	};
 
+	const floatingOneTap = new VKID.FloatingOneTap();
```

Comment:
floatingOneTap.on(VKID.FloatingOneTapInternalEvents.LOGIN_SUCCESS, async (payload: ILoginSuccessPayload) => { ... })

can you add success login event handler witch has handle vkauth request and and redirecting on profitMoney page in 200 code case?